api/services/search-service.js

```diff
@@ -2,39 +2,39 @@
 
 var Client = require('node-rest-client').Client;
 var client = new Client();
-// const GRIDDY_BASE = "http://griddy.org/api";
-const GRIDDY_BASE = "http://localhost:3001/griddy"; // MOCK URL
+
+var today = new Date();
+var year = today.getFullYear();
+const MONTH = "9";
+const UOFT_BASE = "https://timetable.iit.artsci.utoronto.ca/api/" + year + MONTH + "/courses?";
+// const UOFT_BASE = "http://localhost:3001/uoft"; // MOCK URL
 
 module.exports = {
     ENDPOINTS: {
-        courseSearch: GRIDDY_BASE + "/course?q=",
-        courseDataSearch: GRIDDY_BASE + "/course?id="
+        courseSearch: UOFT_BASE,
+        courseDataSearch: UOFT_BASE + "/courses?org=&code="
     },
 
-    sendQueryToGriddy: function (endpoint, params, onRequestCompleted) {
+    sendQueryToUofT: function (endpoint, params, onRequestCompleted) {
         client.get(endpoint + params, function (data) {
             onRequestCompleted(data);
         });
     },
 
     parseCourseSearchResults: function (data) {
```

Comment:
The code for this function should be left as it was ... your code goes in the `parseCourseData` function.